tests/test262-esnext-excludelist.xml

```diff
@@ -2927,23 +2926,66 @@
   <test id="intl402/supportedLocalesOf-unicode-extensions-ignored.js"><reason></reason></test>
   <test id="language/arguments-object/10.6-13-a-2.js"><reason></reason></test>
   <test id="language/arguments-object/10.6-13-a-3.js"><reason></reason></test>
-  <test id="language/arguments-object/arguments-caller.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-2.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-4.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-delete-2.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-delete-4.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-nonwritable-3.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-nonwritable-4.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-strict-delete-2.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/mapped-arguments-nonconfigurable-strict-delete-4.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-descriptors-basic.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-descriptors-set-value-by-arguments.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-descriptors-set-value-with-define-property.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-descriptors-with-param-assign.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-nonenumerable-nonwritable-descriptors-set-by-arguments.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-nonenumerable-nonwritable-descriptors-set-by-param.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-nonwritable-descriptors-set-by-arguments.js"><reason></reason></test>
-  <test id="language/arguments-object/mapped/nonconfigurable-nonwritable-descriptors-set-by-param.js"><reason></reason></test>
+  <test id="language/arguments-object/cls-decl-async-private-gen-meth-args-trailing-comma-multiple.js"><reason></reason></test>
+  <test id="language/arguments-object/cls-decl-async-private-gen-meth-args-trailing-comma-null.js"><reason></reason></test>
+  <test id="language/arguments-object/cls-decl-async-private-gen-meth-args-trailing-comma-single-args.js"><reason></reason></test>
+  <test id="language/arguments-object/cls-decl-async-private-gen-meth-args-trailing-comma-spread-operator.js"><reason></reason></test>
```

Comment:
These tests are already in the exclude list in the "ESNext stage 3 proposals: class fields, private methods and static class features" block at the end.